lib/scripts/sign-apk.js

```diff
@@ -1,8 +1,7 @@
-
 import { androidHelpers } from 'appium-android-driver';
 import { asyncify } from 'asyncbox';
 import path from 'path';
-import packageJson from '../package.json';
+import packageJson from '../../../package.json';
```

Comment:
should find-root be used instead?

Reply:
I think it's okay to leave it as-is. `find-root` is for the case where it's uncertain where package.json will be